### PR TITLE
Length of linked list #231

### LINKED ISSUE #213: Length of a Linked List
Include the following two functions for determining the length of a linked list in these codes (https://github.com/jainaman224/Algo_Ds_Notes/tree/master/Linked_List)
- lengthIterative - This function should be implemented iteratively.
- lengthRecursive - This function should be implemented recursively.

Note : please specify the language in which you are interested before proceeding.

### PR DESCRIPTION
**Pull request by ravireddy07**
Fixes #213 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Length of Linked List in C
- Added Length of Linked List in Java

#### Languages Used:
- C
- Java

#### Files Added:
- LengthOfLinkedList.java
- LengthOfLinkedList.c

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Linked_List/LengthOfLinkedList.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+/*
+This code is written for Char to store in LinkedList, if you want to store numericals or any other datatype,
+just replace relevent data type in place of char in class Node.
+*/
+
+struct Node {
+	char data;
```
Comment (jainaman224) [inline Linked_List/LengthOfLinkedList.c]: 4 space indentation is not followed.

File: Linked_List/LengthOfLinkedList.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	pushing_data_to_LL(&head, 'C');
+	pushing_data_to_LL(&head, '2');
+	pushing_data_to_LL(&head, '0');
+	pushing_data_to_LL(&head, '2');
+	pushing_data_to_LL(&head, '0');
+	
+	printf("The Length of Linkedlist is: %d", gettingLength(head));
+	return 0;
+} 
+
```
Comment (jainaman224) [inline Linked_List/LengthOfLinkedList.c]: Sample output is absent.

File: Linked_List/LengthOfLinkedList.c [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+	pushing_data_to_LL(&head, 'S');
+	pushing_data_to_LL(&head, 'S');
+	pushing_data_to_LL(&head, 'O');
+	pushing_data_to_LL(&head, 'C');
+	pushing_data_to_LL(&head, '2');
+	pushing_data_to_LL(&head, '0');
+	pushing_data_to_LL(&head, '2');
+	pushing_data_to_LL(&head, '0');
+	
+	printf("The Length of Linkedlist is: %d", gettingLength(head));
```
Comment (jainaman224) [inline Linked_List/LengthOfLinkedList.c]: Check length at multiple instances.

**Comment by ravireddy07** [comment]
Sir, i've updated the code with required or mentioned changes. I've pushed also, but nothing showing here. I've created another PR #2222

**Pull request closed without merging**